packages/react/src/stories/ic-accordion.stories.mdx

```diff
@@ -522,12 +522,14 @@ export const Dialog = () => {
 <Canvas>
   <Story name="Slotted headings">
       <IcAccordionGroup>
-  <h1 slot="group-title">Group title</h1>
-  <IcAccordion>
-    <h2 slot="heading">Heading</h2>
-    Text
-  </IcAccordion>
-  </IcAccordionGroup>
+        <h1 slot="group-title">Group title</h1>
```

Comment:
Though the heading is showing as expected in the Cypress tests, it might be worth making this same change to the relevant test data in the IcAccordionTestData.tsx file just for consistency.